src/components/Footer.jsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+
+export const Footer = () => {
+  return (
+    <div>
+      <div className="w-full h-px border-gray-700 border-t inset-y-0 mb-5" />
+      <div className="text-violet-light text-opacity-60 text-xs mb-5">
+        © 2022 Nadiia Ridko. All rights reserved.
```

Comment:
You can also use `new Date().getFullYear()` to automatically display the current year in the copyright notice ;) 
```suggestion
        © {new Date().getFullYear()} Nadiia Ridko. All rights reserved.
```